app/controllers/account_activations_controller.rb

```diff
@@ -0,0 +1,14 @@
+class AccountActivationsController < ApplicationController
+  def edit
+    user = User.find_by email: params[:email]
```

Comment:
tách hẵng việc tìm user ra 1 method và dùng action filter để call cho edit nhé em
nhớ handle user not found